Assignment6.Rmd

```diff
@@ -67,4 +69,11 @@ table(M2$certified, M2$predict1)
 
 table(M2$certified, M2$predict2)
```

Comment:
Good work, it is a good idea to explain your reasoning though, rather than simply stating the answer.